src/Faithlife.WebRequests/WebServiceRequest.cs

```diff
@@ -113,8 +114,8 @@ protected override void OnWebRequestCreated(HttpRequestMessage request)
 			return true;
 		}
 
-		static readonly ReadOnlyCollection<HttpStatusCode> s_defaultAcceptedStatusCodes = ListUtility.CreateReadOnlyCollection(HttpStatusCode.OK, HttpStatusCode.Created);
-		static readonly ReadOnlyCollection<HttpStatusCode> s_defaultAcceptedStatusCodesWithRedirect = ListUtility.CreateReadOnlyCollection(HttpStatusCode.OK, HttpStatusCode.Created,
-			HttpStatusCode.Moved, HttpStatusCode.Redirect, HttpStatusCode.RedirectMethod,  HttpStatusCode.RedirectKeepVerb );
+		static readonly IReadOnlyList<HttpStatusCode> s_defaultAcceptedStatusCodes = new[] { HttpStatusCode.OK, HttpStatusCode.Created };
+		static readonly IReadOnlyList<HttpStatusCode> s_defaultAcceptedStatusCodesWithRedirect = new[] { HttpStatusCode.OK, HttpStatusCode.Created,
+			HttpStatusCode.Moved, HttpStatusCode.Redirect, HttpStatusCode.RedirectMethod,  HttpStatusCode.RedirectKeepVerb };
```

Comment:
Thoughts on using a "real" ReadOnlyCollection for values exposed as public static properties?

Obviously much more important in the .NET Framework where malicious code could cast and mutate them, but wondering if we should have a similar guideline?

Reply:
Given that this is public utility code, it's probably better to be on the safe side.